l10n.js

```diff
@@ -979,7 +979,9 @@ document.webL10n = (function(window, document, undefined) {
     }
 
     // translate element itself if necessary
-    translateElement(element);
+    if (element.nodeType != 11) {
```

Comment:
Use `.nodeType === 1` instead. Text nodes (nodeType 3), documents (nodeType 9?), comments, etc. don't support getAttribute either.

And please squash your commits (https://help.github.com/articles/about-git-rebase/).